tests/csm/rest/test_csm_load.py

```diff
@@ -409,6 +407,15 @@ def test_44790(self):
         assert resp.status_code == HTTPStatus.OK, "List IAM user failed."
         user_data = resp.json()
         self.log.info("Step 1: List user response : %s", user_data)
+        existing_user = len(user_data['users'])
+        self.log.info("Existing iam users count: %s", existing_user)
+        self.log.info("Max iam users : %s", rest_const.MAX_IAM_USERS)
+        new_iam_users = rest_const.MAX_IAM_USERS - existing_user
+        self.log.info("New users to create: %s", new_iam_users)
+
+        self.log.info("Step 2: Create users in parallel")
+        result = self.csm_obj.create_multi_iam_user_loaded(new_iam_users, existing_user)
+        assert result, "Unable to create users"
```

Comment:
without this assert, test is passing

Reply:
[2022-09-07 07:52:32] [MainThread] [INFO  ] [test_csm_load.py: 124]: CSM user deleted []
[2022-09-07 07:52:32] [MainThread] [INFO  ] [test_csm_load.py: 138]: Done: Teardown completed.
======================================================================== short test summary info ========================================================================
PASSED tests/csm/rest/test_csm_load.py::TestCsmLoad::test_44790
=============================================================== 1 passed, 4 warnings in 740.51s (0:12:20) ===============================================================
(virenv) [root@ssc-vm-g2-rhev4-1909 cortx-test]#